packages/application-config/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://unpkg.com/@commercetools-frontend/application-config/schema.json",
+  "title": "JSON schema for Custom Application configuration files",
+  "type": "object",
+  "definitions": {
```

Comment:
The definitions here are "reusable" schema properties that you can reference within the schema.